features/collection/thresholding/impl/src/main/java/org/opennms/netmgt/threshd/AbstractThresholdEvaluatorState.java

```diff
@@ -155,7 +155,7 @@ private void fetchState() {
         // If both of those conditions are false, then we must be on a standalone instance of OpenNMS and have the state
         // already in memory so there is no need to fetch it
         if (!isDistributed() && !firstEvaluation) {
-            fetchState();
+            return;
```

Comment:
Note: this change is just due to screwing up the merge. Just fixed it in this commit since it will all get squashed anyway.